api/v1alpha3/types.go

```diff
@@ -294,6 +298,13 @@ func (s Subnets) FilterByZone(zone string) (res Subnets) {
 	return
 }
 
+// CNISpec defines configuration for CNI
+type CNISpec struct {
+	// IngressRules specify rules to apply to control plane and worker node security groups
+	// The source for the rule will be set to control plane and worker security group IDs
```

Comment:
Godoc/OpenAPI docs don't respect the whitespace. I recommend ending sentences with periods. 😄